src/table.ts

```diff
@@ -327,11 +319,12 @@ class Table {
    *     const apiResponse = data[0];
    *   });
    */
-  deleteRows(keys: Key[]): CommitPromise;
-  deleteRows(keys: Key[], callback: s.CommitCallback): void;
-  deleteRows(keys: Key[], callback?: s.CommitCallback): CommitPromise|void {
+  deleteRows(keys: Key[], callback?: BasicCallback): Promise<BasicResponse>|void {
```

Comment:
Pretty sure this does not return an `IOperation`, unless `BasicResponse` has a typo?

Reply:
Follow up to this, I'm noticing `BasicCallback` and `BasicResponse` being used on most crud methods, rather than calling them out can we just make a point to fix them across the board?

Reply:
I'll fix them across the board 👍